src/FunctionalTest/Smo/ScriptingTests/PartitionFunction_SmoTestSuite.cs

```diff
@@ -71,20 +71,21 @@ public void ScriptCreateWithDbNullRangeCorrectly_PartitionFunction()
             ExecuteFromDbPool(db =>
             {
                 // Create a Partition Function using SMO
-                var partitionFunction = new _SMO.PartitionFunction(db, "PF_NULL_RANGE_TEST");
-                partitionFunction.RangeType = _SMO.RangeType.Left;
+                var partitionFunction = new _SMO.PartitionFunction(db, "PF_NULL_RANGE_TEST")
+                {
+                    RangeType = _SMO.RangeType.Left,
+                    RangeValues = new object[] { DBNull.Value, 10, 100 }
+                };
                 partitionFunction.PartitionFunctionParameters.Add(new _SMO.PartitionFunctionParameter(partitionFunction, _SMO.DataType.Int));
-                partitionFunction.RangeValues = new object[] { DBNull.Value, 10, 100 };
                 partitionFunction.Create();
 
                 var scripter = new _SMO.Scripter(db.Parent);
                 scripter.Options.ScriptForCreateOrAlter = true;
                 scripter.Options.EnforceScriptingOptions = true;
                 scripter.Options.ContinueScriptingOnError = true;
                 var scripts = scripter.EnumScript(partitionFunction).ToArray();
-                Assert.AreEqual(scripts.Length, 1);
-                var result = scripts[0];
-                Assert.IsTrue(result.Contains("VALUES (NULL, 10, 100)"));
+                Assert.That(scripts.Length, Is.EqualTo(1), "Unexpected script count. Expected a single script.");
+                Assert.IsTrue(scripts[0].Contains("VALUES (NULL, 10, 100)"));
```

Comment:
```c# 
Assert.That(scripts[0], Contains.Substring("VALUES (NULL, 10, 100)"), "Incorrect script for null valued function");
```

We've adopted the nUnit constraints the past couple years because they provide very rich log output when they fail and are very expressive as written. The reason all the code doesn't have them is just that SMO is 20+ years old and we haven't tried to retrofit all of it to new styles. We would like all new code to follow the styles defined in .editorconfig and to use the NUnit constraints for asserts.
thx